demisto_sdk/commands/common/constants.py

```diff
@@ -407,6 +415,9 @@ class FileType(Enum):
 PACKS_PRE_PROCESS_RULES_DIR_REGEX = fr'{PACK_DIR_REGEX}/{PRE_PROCESS_RULES_DIR}'
 PACKS_PRE_PROCESS_RULES_JSON_REGEX = fr'{PACKS_PRE_PROCESS_RULES_DIR_REGEX}/(?:preprocessrule-)?([^/]+)\.json'
 
+PACKS_LISTS_DIR_REGEX = fr'{PACK_DIR_REGEX}/{LISTS_DIR}'
+PACKS_LISTS_JSON_REGEX = fr'{PACKS_LISTS_DIR_REGEX}/(?:list-)?([^/]+)\.json'
```

Comment:
Why do we need the list- as a possible for regex? just catch everything inside the dir